scripts/move_linearly.py

```diff
@@ -29,6 +31,7 @@
     time.sleep(args.wait)
 
     start_time = time.time()
+    hardware.both_axes.current_location = [0, 0]
```

Comment:
Yep rename and comment to show this is deliberate rather than a missed replacement.